backend/src/ml_space_lambda/data_access_objects/app_configuration.py

```diff
@@ -237,10 +237,10 @@ def to_dict(cluster_size_list: list):
 
     @staticmethod
     def extract_cluster_sizes(cluster_size_list: list):
-        # Given a list of cluster sizes, parse them into a list of ClusterSize objects
+        # Given a list of cluster sizes, parse them into a list of ClusterType objects
         cluster_object_list = []
         for size in cluster_size_list:
```

Comment:
Depending on the answer to my other comment this method is still calling stuff `size` a lot that might want to be changed to type for consistency.

Reply:
Yeah looks like there's a lot of places that reference clusterSize, will fix all of those in this file